tests/media/report/test_raw_pdf.py

```
@@ -0,0 +1,69 @@
+
+        self.raw_pdf_file = get_default_egd_pdf()
+        self.raw_pdf_file.save()
+
+    def test_raw_pdf_file_creation(self):
+        self.assertIsInstance(self.raw_pdf_file, RawPdfFile)
+        logger.warning(f"Raw PDF file created: {self.raw_pdf_file.file.name}")
+        
+        raw_pdf_file = self.raw_pdf_file
+        center = raw_pdf_file.center
+        text = raw_pdf_file.text
+        sensitive_meta  = raw_pdf_file.sensitive_meta
```

Ruff (0.8.2)

Redefinition of unused `sensitive_meta` from line 12

(F811)